hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/StreamSerializerAdapter.java

```diff
@@ -63,10 +63,7 @@ public Serializer getImpl() {
 
     @Override
     public String toString() {
-        final StringBuilder sb = new StringBuilder("SerializerAdapter{");
-        sb.append("serializer=").append(serializer);
-        sb.append('}');
-        return sb.toString();
+        return "SerializerAdapter{" + "serializer=" + serializer + '}';
```

Comment:
`SerializerAdapter` vs. `StreamSerializerAdapter`? 

`"SerializerAdapter{" + "serializer="` could be merged.